### ISSUE TITLE
Change name to: grpc-debug-interceptor

### ISSUE THREAD

**Issue by jrapoport** [jrapoport/grpc-web-devtools#8]
I'm thinking to change the name of the repo to avoid confusion in advance of publishing to Chrome and am seeking comment. More so on **if** this should happen and not what it's called.

**Comment by jrapoport** [comment]
@borissmidt

**Comment by borissmidt** [comment]
Yes it might be a good idea to have a new name otherwise you get a conflict with the original author.

**Comment by jrapoport** [comment]
I'll move the repo to grpc-debug-interceptor unless there is a better suggestion.

**Comment by borissmidt** [comment]
grpc-web-debug-intereptor since it is meant for grpc web.
